src/frontend/admin/src/hooks/useCopyToClipboard.tsx

```diff
@@ -0,0 +1,22 @@
+import { useIntl } from "react-intl";
+import { useSnackbar } from "notistack";
+import { commonTranslations } from "@/translations/common/commonTranslations";
+
+export const useCopyToClipboard = () => {
+  const intl = useIntl();
+  const snackbar = useSnackbar();
+  return (str?: string) => {
```

Comment:
From my point of view, string should be mandatory here. We should not display the copy to clipboard action if there is nothing to copy.